extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -204,8 +207,9 @@ private synchronized void refreshTimeDate() {
 
             Calendar calendar = Calendar.getInstance(timeZone, locale);
             calendar.setTimeInMillis(networkTimeInMillis);
+            ZonedDateTime zoned= ZonedDateTime.of(LocalDateTime.now(),timeZone.toZoneId());

```

Comment:
your formatter should have put a space in front of the equals sign and behind the comma!